blocks/components/editable/index.js

```diff
@@ -296,12 +299,7 @@ export default class Editable extends wp.element.Component {
 		if ( focus ) {
 			element = [
 				<Fill name="Formatting.Toolbar" key="fill">
-					<Toolbar
-						controls={ formattingControls.map( ( control ) => ( {
-							...control,
-							onClick: () => this.toggleFormat( control.format ),
-							isActive: this.isFormatActive( control.format )
-						} ) ) } />
+					<FormatToolbar focusPosition={ this.state.focusPosition } formats={ this.state.formats } onChange={ this.changeFormats } />
```

Comment:
I do like this link design (though the specific buttons are a bit quickly put together):

![links](https://cloud.githubusercontent.com/assets/1204802/25498925/02c6fe94-2b8a-11e7-9264-5e98b6379f21.png)

@iseulde should be in the loop on the link work we're doing here, because IMO links work better in the current WordPress editor than anywhere else, and she worked a lot on them. Having the dialog under as opposed to over was for a reason, though I can't recall which.

Also, here's an admittedly suboptimal design for when you are linking inside captions:

![screen shot 2017-04-27 at 20 41 49](https://cloud.githubusercontent.com/assets/1204802/25498964/2a167d08-2b8a-11e7-9503-7583f5e06408.png)

Still thinking about that one, but it's at least one argument for having links under.